products/sle/main.pm

```diff
@@ -103,6 +103,10 @@ sub cleanup_needles {
         unregister_needle_tags('ENV-JEOS-1');
     }
 
+    if (!is_microos) {
+        unregister_needle_tags('ENV-MICROOS');
```

Comment:
Shouldn't we use same naming convention, so variable names are `ENV-MICROOS-1` and `ENV-MICROOS-0`?
In general there is not much use in `ENV-MICROOS-0`, but we have it for all other variables...

Reply:
I didn't find useful using the `-0` or `-1` in this case, since it lacks meaning, but I will do it to match other tags... I will need to rename also it in the needles.

Reply:
What about 'ENV-DISTRI-microos` ?

Reply:
That will do the trick and we have code in openSUSE which won't require special condition every time we add new distribution tag:

`unregister_needle_tags("ENV-DISTRI-$distri") unless check_var('DISTRI', $distri);`

Reply:
done